File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_python/p03292/p03292_num3_parsed.py
# program to find the minimum total cost of completing the tasks

# function to calculate the minimum total cost
def min_total_cost(tasks):
    # sort the tasks in ascending order
    sorted_tasks = sorted(tasks)

    # calculate the minimum cost
    min_cost = sorted_tasks[2] - sorted_tasks[0]

    return min_cost

# take input from user
input_str = input()
tasks = list(map(int, input_str.split()))

# calculate the minimum total cost
cost = min_total_cost(tasks)

# print the output
print(cost)

File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_cpp/p03292/p03292_num3_parsed.cpp
#include <iostream>
#include <cmath>

int main() {
    // Read input
    int A[3];
    for (int i = 0; i < 3; i++) {
        std::cin >> A[i];
    }

    // Calculate the minimum total cost
    int minTotalCost = std::abs(A[0] - A[1]) + std::abs(A[1] - A[2]);
    minTotalCost = std::min(minTotalCost, std::abs(A[0] - A[2]) + std::abs(A[2] - A[1]));
    minTotalCost = std::min(minTotalCost, std::abs(A[1] - A[0]) + std::abs(A[0] - A[2]));

    // Print the minimum total cost
    std::cout << minTotalCost << std::endl;

    return 0;
}